pages/opportunities/[slug].tsx

```diff
@@ -83,16 +80,16 @@ const Page: NextPage<PageProps> = (props) => {
                     </a>
                   )}
               </S.OpportunityMetaRow>
-              <OpportunityMetaRow>
+              <S.OpportunityMetaRow>
                 <TimeIcon />
                 <Body>{opportunity.timeRequirements}</Body>
-              </OpportunityMetaRow>
+              </S.OpportunityMetaRow>
               <S.OpportunityOwnerWrapper>
                 <Body>Kontaktní osoba</Body>
                 <S.OwnerWrapper>
                   <S.OwnerImage src={owner.profilePictureUrl} />
                   <div>
-                    <OwnerName>{owner.name}</OwnerName>
+                    <OwnerContact email={owner.email} name={owner.name} />
                     <BodySmall>{parentProject.name}</BodySmall>
```

Comment:
Nechceme sem ten email napsat misto nazvu projektu? Podle me tam ten projekt byl jenom jako placeholder, aby to nebylo prazdne misto. (v puvodni verzi tam byl prave email, ktery jsme odstranli).

Reply:
To bude vypadat mnohem líp, díky, předělám!